data/flexibleForms/forms.types.ts

```diff
@@ -92,11 +92,13 @@ export interface Submission {
   createdAt: string;
   residents: Resident[];
   workers: Worker[];
-  editHistory: {
-    worker: Worker;
-    editTime: string;
-  }[];
+  editHistory: Revision[];
   submissionState: 'In progress' | 'Approved' | 'Discarded' | 'Submitted';
   formAnswers: FlexibleAnswers;
   tags?: string[];
 }
+
+export interface Revision {
+  worker: Worker;
+  editTime: string;
```

Comment:
Shouldn't the type be `Date`?

Reply:
it comes in as a string from the api - we convert it to a date just before we display it